### ISSUE TITLE
Handle Underline in Markdown

### ISSUE THREAD

**Issue by irmerk** [accordproject/web-components#28]
# To Be Done Instead of OR After Issue accordproject/markdown-editor#97

Since transitioning to [utilize the `markdown-transform`](https://github.com/accordproject/markdown-editor/releases/tag/v0.6.0), we will need to take a step back from supporting underline for the time being. Markdown doesn't natively support underline, so we need to rethink how we serialise this. 

We can then experiment with using `html` tags for underline.

**Comment by d-e-v-esh** [comment]
@irmerk Can I look into this?

**Comment by irmerk** [comment]
@d-e-v-esh I believe this may actually require thinking/work in https://github.com/accordproject/markdown-transform first. It could be worth opening an Issue there to discuss how to (or if to) handle underline in markdown.

**Comment by d-e-v-esh** [comment]
@irmerk Yes. I think we can open another issue on this. Underlining text seems like it could be important for people dealing with contracts. Should I open another issue on this? This one will get closed?

**Comment by irmerk** [comment]
If you could open one in `markdown-transform` and link this one, we can put a pause on this one until that new one is resolved.
